ClosedXML/Excel/Drawings/IXLPicture.cs

```diff
@@ -57,21 +58,21 @@ public interface IXLPicture : IDisposable
         /// <returns>A created copy of the picture.</returns>
         IXLPicture Duplicate();
 
-        Point GetOffset(XLMarkerPosition position);
+        SKPoint GetOffset(XLMarkerPosition position);
 
-        IXLPicture MoveTo(Int32 left, Int32 top);
+        IXLPicture MoveTo(float left, float top);
```

Comment:
i am not entirely sure if that's a good idea. It seems to me that this is something that should be checked against the OpenXML standard. why did you change this to float?

Reply:
I just looked it up and found that coordinates are of the following type:
`  <xsd:simpleType name="ST_CoordinateUnqualified">
    <xsd:restriction base="xsd:long">
      <xsd:minInclusive value="-27273042329600"/>
      <xsd:maxInclusive value="27273042316900"/>
    </xsd:restriction>`

on the other hand when this is serialized everything walks through ConvertToEnglishMetricUnits to make EMU out of pixels. So the only "problem" is maybe some loss of precision when values of "number of pixels" (that is what left and top are basically) are very high. Which on the other hand is quite unrealistic and especially unproblematic.

So bottom line: this is not ideal but Ok for production use imho